pkg/operator/utils/version_check_test.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestCheckForMinimumPatchedVersion(t *testing.T) {
+	tests := []struct {
+		name             string
+		vSphereVersion   string
+		buildNumber      string
+		meetsRequirement bool
+	}{
+		{
+			name:             "when vSphere version meets minimum 7 series requirement",
+			vSphereVersion:   "7.0.3",
+			buildNumber:      "23788037",
+			meetsRequirement: true,
+		},
+		{
+			name:             "when vSphere version meets minimum 8 series requirement",
+			vSphereVersion:   "8.0.2",
+			buildNumber:      "23504391",
+			meetsRequirement: true,
+		},
```

Comment:
just for the fun, please add a test with v9. I think it will work just fine.

Reply:
it depends on build number we specify for 9 series in current code. It will not pass for any random build number fwiw. :/

Reply:
So, when v9 is introduced, we need to update the code to add support for it? That's unfortunate.

Reply:
I have added code with the assumption that anything higher than 8 will automatically be supported.